lib/engine/step/g_18_co/tracker.rb

```diff
@@ -77,6 +79,25 @@ def all_new_paths_have_new_exits?(brand_new_exit_junctions)
           brand_new_exit_junctions.size >=
             brand_new_exit_junctions.sum { |_bnp, new_exit_junctions| new_exit_junctions.size }
         end
+
+        def pending_token(entity)
+          @round.pending_tokens.find { |pt| pt[:entity] == entity }
```

Comment:
doesn't look like you use pending_token anywhere, you should switch to any? instead of find

Reply:
Tracker is an include and is used in those files.